app.js

```diff
@@ -75,6 +75,9 @@
     // Expose everything else as static files
     app.use(express['static'](options.directory));
 
-    // Finally, open the HTTP server
-    app.listen(options.port);
-}());
\ No newline at end of file
+    // Finally, open the HTTP server and log the instance to the console
+    app.listen(options.port, function() {
+        console.log('Open MCT application running at localhost:' + options.port)
+    });
+ 
+    }());
```

Comment:
There's some extra indentation on this last line (which closes the IIFE started on line 10)